blockchain/fullblocktests/generate.go

```diff
@@ -1347,7 +1347,7 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 
 		prevBlock := g.Tip()
 		spend := chaingen.MakeSpendableOut(prevBlock, 1, 0)
-		ticketPrice := dcrutil.Amount(b.STransactions[5].TxOut[0].Value)
+		ticketPrice := dcrutil.Amount(g.CalcNextRequiredStakeDifficulty())
```

Comment:
This is not correct. It will calculate the ticket price for the next block, not the current one.